src/components/OpenSource/Contributions.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import styled from 'styled-components'
+import BlueBackground from '../Common/BlueBackground'
+import ContributionsCopy from './ContributionsCopy'
+import { Grid, Row, Col } from '../grid'
+import { Repo } from '../Repo'
+import { generate } from 'shortid'
+import StyledLink from '../Common/StyledLink'
+import Image from '../Common/Image'
+import { BodyPrimary } from '../Typography'
+
+const Wrapper = styled(BlueBackground)`
+  padding: ${props => props.theme.spacing[4]} 0;
+`
+
+const StyledBodyPrimary = styled(BodyPrimary)`
+  padding: ${props => props.theme.spacing[3.5]} 0;
+`
+
+const RepoWrapper = styled.div`
+  padding-bottom: ${props => props.theme.spacing[3]};
+`
+
+const StyledImage = styled(Image)`
+  max-width: 54px;
+`
+
+const Contributions = ({
```

Comment:
Spacing between components is a bit off. Needs `24px` between image & title + `24px` between title and body.

Designs
![image](https://user-images.githubusercontent.com/15656538/58712624-d444a380-83b8-11e9-8f2b-b87b44afbf32.png)

This branch
![image](https://user-images.githubusercontent.com/15656538/58712585-c42cc400-83b8-11e9-9b4f-d667d505e716.png)